### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by UnicodeAlt255**

### CODE REVIEW

**Comment by UnicodeAlt255** [comment]
Dang, I thought it would be a quick PR, but I'll continue to add commits as I find them, and push them later.

**Comment by hoodie** [comment]
Oh man, I'm really not good add spelling

**Comment by hoodie** [comment]
Could you squash them though?

**Comment by UnicodeAlt255** [comment]
> Oh man, I'm really not good *add* spelling

😂

**Comment by hoodie** [comment]
yeah, that was kinda on purpose though :laughing:

File: src/hints.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//! `NotificationHints` allow to pass extra information to the server.
+//! `NotificationHints` allow you to pass extra information to the server.
```
Comment (hoodie) [inline src/hints.rs]: not really a typo, but ok

File: src/xdg.rs [hunk 2]
```diff
@@ -105,7 +105,7 @@ impl Deref for NotificationHandle {
     }
 }
 
-/// Allow to easily modify notification properties
+/// Allow you to easily modify notification properties
```
Comment (hoodie) [inline src/xdg.rs]: This should probably be "Allows to easily modify..." but, ok

**Pull request merged**